CONTRIBUTING.md

```diff
@@ -158,18 +162,25 @@ opam switch create 4.06.1
 opam switch 4.06.1
 eval `opam config env`
 
-opam install js_of_ocaml.3.5.1
+opam install js_of_ocaml.3.6.0
+```
+
+Additionally, make sure to adapt the `scripts/repl.js` file to use the `js_of_ocaml` binary instead (otherwise you'll continue using the slower bytecode version):
+
+```diff
+- e(`${OCAMLRUN} ${JSOO} compile jsc.byte ${jsooFlag}-o exports.js`);
++ e(`js_of_ocaml compile jsc.byte ${jsooFlag}-o exports.js`);
 ```
 
-### Build the Bundle
+### Building the Bundle
 
-The entry point of the JSOO bundle is located in `jscomp/main/jsoo_main.ml` and the script for running JSOO can be found in `scripts/repl.js`. A full clean build can be done like this:
+The entry point of the JSOO bundle is located in `jscomp/main/jsoo_refmt_main.ml` and the script for running JSOO can be found in `scripts/repl.js`. A full clean build can be done like this:
 
```

Comment:
I just removed the js part of refmt a couple of weeks ago (i.e, js_refmt_main.ml). 
Since the playground is not essential part of keep making reason working, I am not convinced to add it back.
- For people who want to learn the conversion from reason to rescript syntax, they can use the CLI tools
```
npx bsc -format xx.re
```
it will print the rescript syntax
- remove the reason syntax for the playground not only bring less maintainance overhead, it makes the playground load faster. On a slow network (my case), it takes a while to load the playground

Reply:
We still have a bunch of production users on ReasonML syntax, so they heavily rely on the playground functionality to convert between ReScript and Reason snippets.

![image](https://user-images.githubusercontent.com/1121889/108319175-47abcb80-71c1-11eb-9154-9de217a9dcb8.png)

^ The toggle seen here uses the vendored refmt code, which is the exact same version as the refmt we ship in the compiler toolchain still.

The last blocking factors for some of our proudction users are "local open" support (some still want infix operators, but I think that is less pressing than local open).

I'd suggest to removing Reason support on the playground when we are 100% sure that we are not interfering with our users workflow. Maybe we should open a topic on the forum discussing this?